cypress/integration/main/keystroke.spec.js

```diff
@@ -9,6 +9,15 @@ const SETTINGS_SLIDE_CLOSE = 'slide-close-settings';
 
 describe('Timer Keys Test', ()=>{
     beforeEach(() => {
+        // unregisters service worker before e2e test

```

Comment:
Could possibly encapsulate this into a function to eliminate code repetition

Reply:
made this into a function in sw.js